packages/relay-test-utils-internal/node.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ * @oncall relay
+ */
+
+'use strict';
+
+const {
+  FIXTURE_TAG,
+  generateTestsFromFixtures,
+} = require('./generateTestsFromFixtures');
```

Comment:
This module uses Node.js APIs. Better to isolate the utils that require a Node environment from those that don't. These helpers are only used in the Babel plugin tests.